README.md

```diff
@@ -9,33 +9,66 @@ This Go package provides a wrapper for the [Luno API](https://www.luno.com/api).
 Please visit [godoc.org](https://godoc.org/github.com/luno/luno-go) for the full
 package documentation.
 
+### Authentication
+
+Please visit the [Settings](https://www.luno.com/wallet/settings/api_keys) page
+to generate an API key.
+
 ### Installation
 
 ```
 go get github.com/luno/luno-go
 ```
 
-### Authentication
-
-Please visit the [Settings](https://www.luno.com/wallet/settings/api_keys) page
-to generate an API key.
-
 ### Example usage
 
+A full working example of this library in action.
+
 ```go
+package main
+
 import luno "github.com/luno/luno-go"
+import (
+  "log"
+  "context"
+  "time"
+)
 
 lunoClient := luno.NewClient()
-lunoClient.SetAuth("api_key_id", "api_key_secret")
+lunoClient.SetAuth("<API_KEY_ID>", "<API_KEY_SECRET>")
 
 req := luno.GetOrderBookRequest{Pair: "XBTZAR"}
-res, err := lunoClient.GetOrderBook(&req)
+ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(10 * time.Second))
+defer cancel()
+
+res, err := lunoClient.GetOrderBook(ctx, &req)
 if err != nil {
   log.Fatal(err)
 }
 log.Println(res)
 ```
 
+Remember to substitute `<API_KEY_ID>` and `<API_KEY_SECRET>` for your own Id and Secret.
+
+It is also recommended to set these as environment variables rather than include them in plaintext. You can do this on `BASH` with:
```

Comment:
Suggest: We recommend using environment variables rather than including your credentials in plaintext. In bash you do so as follows: